corehq/apps/app_manager/views/cli.py

```diff
@@ -25,6 +26,7 @@
 
 @json_error
 @api_auth
+@api_throttle
```

Comment:
This is the one I was unsure about. I think this is used by the CLI, and maybe the phone (with that special feature that showed users a list of available apps), but I don't think either of those are in wide use so thought it made sense.